docs/current_docs/cookbook/snippets/cache-dependencies/python/poetry/main.py

```diff
@@ -0,0 +1,31 @@
+from typing import Annotated
+
+import dagger
+from dagger import Doc, dag, function, object_type
+
+
+@object_type
+class MyModule:
+    @function
+    def build(
+        self, source: Annotated[dagger.Directory, Doc("Source code location")]
+    ) -> dagger.Container:
+        """Build an application using cached dependencies"""
+        return (
+            dag.container()
+            .from_("python:3.11")
+            .with_directory("/src", source)
+            .with_workdir("/src")
+            .with_mounted_cache("/root/.cache/poetry", dag.cache_volume("poetry_cache"))
```

Comment:
I might be wrong because I lack of experience with poetry but [according to the doc](https://python-poetry.org/docs/configuration/#cache-dir), poetry default cache dir is `$HOME/.cache/pypoetry`.

```suggestion
            .with_mounted_cache("/root/.cache/pypoetry", dag.cache_volume("poetry_cache"))
```

Let me know what's the correct answer :D

Reply:
Yes, that's correct.